### ISSUE TITLE
Custom Sound

### ISSUE THREAD

**Issue by jrahma** [edsnider/LocalNotificationsPlugin#46]
Hi,

Is sending local notification with custom sound possible using the plugin?

If not then it will be great if this will be added as a feature request.

Thanks,
Jassim

**Comment by Mikilll94** [comment]
Yes. That would a great feature. I think this is easy to implement.

**Comment by Mikilll94** [comment]
Just use (on Android):

`
builder.SetSound (RingtoneManager.GetDefaultUri(RingtoneType.Alarm));
`
